Assets/Scripts/Bee/BeeTutorialGo.cs

```diff
@@ -21,7 +22,8 @@ public void OnEnable()
     private void DoLeftButtonOnePressed(object sender, ControllerInteractionEventArgs e)
     {
         buttonXHighlight.SetActive(false);
-        gameObject.SetActive(false);
+        gameObject.GetComponent<MeshRenderer>().enabled = false;
+        beeLeftControls.SetActive(true);
```

Comment:
probably easier for merging to do a scene transition instead of putting it in the same scene